cobbler/items/profile.py

```diff
@@ -266,13 +266,17 @@ def enable_ipxe(self) -> bool:
         return self._resolve("enable_ipxe")
 
     @enable_ipxe.setter
-    def enable_ipxe(self, enable_ipxe: bool):
+    def enable_ipxe(self, enable_ipxe: Union[str, bool]):
         r"""
         Setter for the ``enable_ipxe`` property.
 
         :param enable_ipxe: New boolean value for enabling iPXE.
         :raises TypeError: In case after the conversion, the new value is not of type ``bool``.
         """
+        if enable_ipxe == enums.VALUE_INHERITED:
+            self._enable_ipxe = enums.VALUE_INHERITED
```

Comment:
You might want to add an ignore comments so that Codacy doesn't think these values are unused.

Reply:
Vulture works via "whitelists" which are essentially empty Python files that use the code manually instead of using a comment. I have attempted this in cobbler-tftp and find it for the scope of this PR not appropriate. Thanks for the explicit mention though!